extensions/wikia/CuratedContent/CuratedContentSpecialController.class.php

```diff
@@ -198,7 +198,7 @@ public function save() {
 		$this->response->setVal( 'status', $status );
 
 		if ( $status ) {
-			wfRunHooks( 'CuratedContentSave' );
+			wfRunHooks( 'CuratedContentSave', [ $sections ] );
```

Comment:
Yes - parameter is not used - but let's add param to mentioned function for completeness.